security/login-controller.php

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Login</title>
+</head>
+<body>
+<a href="index.php"> Take me home, country roads ... </a> <br>
+<?php
+include "db-connection.php";
+$conn = openCon();
+
+$sql = "SELECT * from person where email = '" . htmlentities($_POST["email"]) . "' and password = '" . htmlentities($_POST["password"]) . "';";
```

Comment:
This is not protected against sql injection. I was able to login with `notpass' OR '1'='1` as the password value. Please check here for a reference implementation: https://github.com/tancheto/WEB2020-php/blob/4a5142909392a6d4ca90e939d13abd33844707a0/hello-security/login-controller.php